core/baseline-core-v1.0-psd01.md

```diff
@@ -4704,6 +4722,34 @@ In the context of BPI interoperability, the BPI export operation MUST provide at
 * Specification of the prover system of the lock commitment
 * A validation URI of the originating BPI that allows a 3rd party to independently verify the lock commitment
 
+[[R137]](#r137) Testability:
+
+Preconditions:
+
+* Two BPI systems are set up: one to export the data and one to import the data.
+* Both BPI systems are properly configured for interoperability.
+* The data in the exporting BPI is in a valid and verifiable state.
+* The BPI export operation is accessible and functional.
+
+Test Steps:
+
+1. Initiate the BPI export operation from the exporting BPI system and review the exported data.
+
+Expected Result:
+1. The State Object is included in the exported data. 
+2. The Zero-Knowledge Proof(s) of Correctness are part of the exported data. 
+3. All public input data for validating the Zero-Knowledge Proofs is provided. 
+4. Private input data for the Zero-Knowledge Proofs is included securely. 
```

Comment:
@KCP1023 to verify a proof private data is not required, just the public inputs, th zkp, the verification key

Reply:
Also, can you remove line 4714 in the requirement? That was included in error.

Reply:
And line 4720